tests/test_h5store.py

```diff
@@ -623,6 +636,88 @@ def set_x(x):
         self.assertIn(self.get_h5store()['x'], set(range(100)))
 
 
+def _read_from_h5store(filename, **kwargs):
+    from signac.core.h5store import H5Store
+    with H5Store(filename, **kwargs) as h5s:
+        list(h5s)
+
+
+@unittest.skipIf(
+    six.PY2 or (six.PY3 and sys.version_info.minor < 4),
+    'tests only implemented for Python > 3.4')
```

Comment:
Do you mean `Python >= 3.4`?

Reply:
Yes, but also I think it should run on 3.4 now, so I'll fix that.